primeorder/src/projective.rs

```diff
@@ -122,7 +122,7 @@ where
         }
 
         let mut q = Self::IDENTITY;
-        let mut pos = C::Uint::BITS as usize - 4;
+        let mut pos = (<Scalar<C> as PrimeField>::NUM_BITS.div_ceil(8) * 8) as usize - 4;
```

Comment:
Actually, now that I think about it, `.div_ceil(4) * 4` would be more accurate, since we need an integer number of window sizes, not bytes.

Reply:
Maybe extract a function for calculating this? It could also be used by a hypothetical `mul_vartime`